convert2rhel/pkgmanager/handlers/dnf/__init__.py

```diff
@@ -86,6 +86,10 @@ def _set_up_base(self):
         # going in with the packages in the transaction.
         self._base._ds_callback = DependencySolverProgressIndicatorCallback()
 
+        # Override the exclude option that is loaded from the config and set it
+        # to empty.
+        self._base.conf.substitutions["exclude"] = []
```

Comment:
I think this should be set after `self._base.read_all_repos()` is called. Otherwise, it raises an exception.

Also, I'm not sure about the name`exclude`, I couldn't find that in the docs. The only one I found and seems like what we want is `excludepkgs` https://dnf.readthedocs.io/en/latest/conf_ref.html#conf-main-options-label

Reply:
The use of `substitutions` is wrongly used here as pointed correctly by @hosekadam. We should rely on `exclude` or `excludepkgs` as well. I'm investigating on why dnf is not loading the settings properly through the config file.

Reply:
The `excludepkgs` is a dnf only param, but either `exclude` or `excludepkgs` can be used. `exclude` is compatible with yum, and since we are pretty much using it everywhere, I think I will keep the `exclude` in the code. 

The substitutions part is really wrong here, but I'm pushing a commit that will fix it.